apps/aecore/test/aecore_sync_SUITE.erl

```diff
@@ -76,31 +74,43 @@ suite() ->
 init_per_suite(Config) ->
     ok = application:ensure_started(erlexec),
     DataDir = ?config(data_dir, Config),
-    TopDir = top_dir(DataDir),
-    Config1 = [{top_dir, TopDir}|Config],
-    make_shortcut(Config1),
+    TopDir = aecore_suite_utils:top_dir(DataDir),
+    Config1 = [{symlink_name, "latest.sync"},
+               {top_dir, TopDir},
+               {test_module, ?MODULE}] ++ Config,
+    aecore_suite_utils:make_shortcut(Config1),
     ct:log("Environment = ~p", [[{args, init:get_arguments()},
                                  {node, node()},
                                  {cookie, erlang:get_cookie()}]]),
-    create_configs(Config1),
-    make_multi(Config1),
+    aecore_suite_utils:create_configs(Config1, #{}, [{add_peers, true}]),
+    aecore_suite_utils:make_multi(Config1),
     Config1.
 
 end_per_suite(Config) ->
-    stop_node(dev1, Config),
-    stop_node(dev2, Config),
-    stop_node(dev3, Config),
+    lists:foreach(
+        fun(Node) ->
+            aecore_suite_utils:stop_node(Node, Config)
+        end,
+        [dev1, dev2, dev3]),
     ok.
 
 init_per_group(two_nodes, Config) ->
     config(two_nodes, Config);
 init_per_group(three_nodes, Config) ->
     Config1 = config(three_nodes, Config),
-    stop_nodes(Config1),
+    lists:foreach(
+        fun(Node) ->
+            aecore_suite_utils:stop_node(Node, Config)
+        end,
+        [dev1, dev2]),
```

Comment:
Why not ensuring that `dev3` node is stopped?

Reply:
This should be refactored. This stops the nodes from the previous run.
I'll fix it